modules/custom/drupalcampmel_core/includes/user.inc

```diff
@@ -30,12 +30,18 @@ function drupalcampmel_core_form_user_register_form_alter(&$form, &$form_state)
       }
     }
 
+    // Add a description
+
+    $form['field_tickets']['und']['#description'] = t("Choose which days you want to attend.");
+    $form['field_tickets']['und']['#title_display'] = "invisible";
+
+
     // Modify form fields; use placeholder attribute instead of title and hide
     // until a ticket is selected.
-    foreach (array('first', 'last') as $field) {
-      $form["field_{$field}_name"][LANGUAGE_NONE][0]['value']['#attributes']['placeholder']      = $form["field_{$field}_name"][LANGUAGE_NONE][0]['value']['#title'];
-      $form["field_{$field}_name"][LANGUAGE_NONE][0]['value']['#title_display']                  = 'invisible';
-      $form["field_{$field}_name"]['#states']['invisible']['#edit-field-tickets .form-checkbox'] = array(
+    foreach (array('first_name', 'last_name', 'diet', 't_shirt_size') as $field) {
```

Comment:
t_shirt_size isn't a textfield, therefore the first two lines in this loop may not be relevant.